tests/Struct/FindologicProductTest.php

```diff
@@ -333,20 +348,29 @@ public function testProductPrices(?Price $price, ?string $exception): void
         }
     }
 
+    public function directIntegrationProvider()
```

Comment:
1. Rename this to `integrationTypeProvider`
2. Refactor this test so the data provider returns the integration type e.g. `API` or `Direct Integration` (don't use the constants, use the direct values).

Reply:
See https://github.com/findologic/plugin-shopware-6/pull/196#discussion_r676517958